CHANGES.md

```diff
@@ -23,6 +23,24 @@ OpenSSL 3.0
 
 ### Changes between 1.1.1 and 3.0 [xx XXX xxxx]
 
+ * Added a library context that applications as well as other
+   libraries can use to form a separate context within which libcrypto
+   operations are performed.
+
+   There are two ways this can be used:
+
+   - Directly, by passing a library context to functions that take
+     such an argument, such as `EVP_CIPHER_fetch` and similar algorithm
+     fetching functions.
+   - Indirectly, by creating a new library context and then assigning
+     it as the new default, with `OPENSSL_CTX_set0_default`.
```

Comment:
The way in which we write the intro above combines with this bullet to suggest that it's okay for an "other library" to use `OPENSSL_CTX_set0_default()`, which is very much not the case.

Reply:
Err...yes it is?? As long as it sets it back again before control returns to the application then that should be fine.

Reply:
So I think we should say something about save/restore here.  An application could perhaps just set and forget, but a library should not.

Reply:
Point received...  I just pushed a fixup that adds a few lines on the matter.

Reply:
@mattcaswell, does that reset the clock?

Reply:
Thank you!